src/styles/index.scss

```diff
@@ -1,3 +1,108 @@
+$day-size: 100px;
+$day-border: 1px;
+$day-gap: $day-border;
+$day-bg: #eee;
+$font-size: 30px;
+$month-length: 31;
+$day-size: 100px;
```

Comment:
The variable `$day-size` is declared multiple times. Consider removing the redundant declarations to improve code clarity and maintainability.